lib/model/message.dart

```diff
@@ -270,6 +270,14 @@ class MessageStoreImpl with MessageStore {
       if (anyMessageFound) {
         for (final view in _messageListViews) {
           view.notifyListenersIfAnyMessagePresent(event.messages);
+          // TODO(#818): Support MentionsNarrow live-updates when handling
+          // @-mentioned flags.
```

Comment:
nit: indent any continuation lines on a TODO comment:
```suggestion
          // TODO(#818): Support MentionsNarrow live-updates when handling
          //   @-mentioned flags.
```

Reply:
(also "@-mentioned flags" reads to me as about flags that were @-mentioned; could say "mention flags" or "@-mention flags")